streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -536,6 +537,11 @@ public class StreamsConfig extends AbstractConfig {
     public static final String ROCKS_DB = "rocksDB";
```

Comment:
IIRC we decided to deprecate the old config, in which case we should make sure to add a `@Deprecated` tag to the config name and these two options. I guess since the doc string is public it should technically also be deprecated...